### PR TITLE
fix: clean-up stories

### PR DESCRIPTION
**Pull request by richburdon**
- [x] Normalize titles.
- [ ] Prep for publishing.

copilot:all

### CODE REVIEW

File: packages/apps/halo-app/src/components/ProgressBar/ProgressBar.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ export type ProgressBarProps = {
   className?: string;
 };
 
+// TODO(burdon): Move to react-ui?
```
Comment (thure) [inline packages/apps/halo-app/src/components/ProgressBar/ProgressBar.tsx:13]: `react-ui` has `Status` which does the same thing, so this component should be deprecated.

File: packages/ui/react-ui-mosaic/src/views/Demo/Echo.stories.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ const Story: FC<{ spaceKey: PublicKey }> = ({ spaceKey }) => {
 };
 
 export default {
-  title: 'Views/Demo',
+  title: 'react-ui-mosaic/Demo',
```
Comment (thure) [inline packages/ui/react-ui-mosaic/src/views/Demo/Echo.stories.tsx:45]:
```suggestion
  title: 'react-ui-mosaic/Echo',
```

**Review by thure — APPROVED** [review]
Looks good to me! Thanks for doing this.

File: packages/apps/halo-app/src/components/ProgressBar/ProgressBar.tsx (hunk 1 above)
Comment (richburdon) [reply packages/apps/halo-app/src/components/ProgressBar/ProgressBar.tsx:13]: @zhenyasav

**Pull request merged**